admin/templates/util/pagination.html

```diff
@@ -3,7 +3,7 @@
     <div class="pagination pagination-lg">
         <span>
             {% if items.has_previous %}
-                <a href="?page={{ items.previous_page_number }}"
+                <a href="?page={{ items.previous_page_number }}&status={{ status }}"
```

Comment:
I'm not sure if this template has implications for non-spam views (maybe pre-reg), but will status always be passed in here?